src/OpenTelemetry.Instrumentation.MySqlData/TracerProviderBuilderExtensions.cs

```diff
@@ -41,7 +41,7 @@ public static class TracerProviderBuilderExtensions
     /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
     public static TracerProviderBuilder AddMySqlDataInstrumentation(
         this TracerProviderBuilder builder,
-        Action<MySqlDataInstrumentationOptions> configure)
+        Action<MySqlDataInstrumentationOptions>? configure)
```

Comment:
We should probably not put `?` to stay consistent with other instrumentation libraries.

Look at this for reference: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.AspNetCore/TracerProviderBuilderExtensions.cs#L59-L71

Reply:
I have discussion with @alanwest some time ago. The consensus is to make it nullable non-optional.

See https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/913

Reply:
Ah okay. Thanks for sharing that.